docs/monthly/2023-10-monthly.md

```diff
@@ -0,0 +1,221 @@
+
+### Kupo x Hydra
+
+Indexers of chain data, like [kupo](https://github.com/CardanoSolutions/kupo),
+are useful to enable lightweight decentralized applications like Hypix. As the
+application state of Hypix, for example, is not only on the Cardano layer 1, but
+in the Hydra head, kupo was enhanced to run "in front of" a `hydra-node` and
+provide the same API for UTxO indexed from a Hydra head. This was a [joint
+effort](https://github.com/CardanoSolutions/kupo/pull/117) between
+[@KtorZ](https://github.com/KtorZ), [@v0d1ch](https://github.com/v0d1ch) and
+[@ch1bo](https://github.com/ch1bo) and has been [released in version 2.7 of
+kupo](https://github.com/CardanoSolutions/kupo/releases/tag/v2.7).
+
+![Kupo indexing data](./img/2023-10-kupo.gif) <small><center>Kupo indexing a Hydra head</center></small>
+
+### Offline-mode Hydra node
+
+[Ledger-only
+Mode](https://projectcatalyst.io/funds/10/f10-development-and-infrastructure/sundae-labs-hydra-ledger-only-mode)
+for Hydra was one of the Catalyst Fund10 projects who got funded and
+work is underway to make it an integral part of Hydra software
+components. A [new
+ADR](https://hydra.family/head-protocol/unstable/adr/28.%20Offline%20mode)
+has been proposed which details the rationale behind this new feature
+and an overview of what it brings and what impacts it will have on the
+hydra-node.
+
+Moreover, implementation is
+[underway](https://github.com/input-output-hk/hydra/pull/1118) and
+should be part of the next release. This will enable running a single
+hydra-node as a kind of "ledger-as-a-service", priming it with some
+UTxO and allowing clients to submit and validate transactions without
+any relationship to Layer 1.
+
+### Voting on Hydra
+
+The
+[Hydra-for-voting](https://github.com/cardano-foundation/hydra-voting-poc)
+project has been started earlier this year as a proof-of-concept to
+explore how a Hydra Head could help improve the voting or rather
+tallying process for Catalyst and other large-scale on-chain
+decision-making process. It's been on pause for a few months due to
+other priorities but it's been revived recently and put to use to
+tally votes for the Cardano Summit's awards recipients.
+
+This development effort lead to a couple new features and fixes:
+* The ability to [submit
+  transaction](https://github.com/input-output-hk/hydra/issues/1111)
+  for L1 using "standard" CBOR-hex format
+* Upgrade to the checked-in [protocol
+  parameters](https://github.com/input-output-hk/hydra/issues/1117)
+  used by demo network which were out of sync with the internal ledger
+  version
+* Discussion on the usage of [Extended payment
+  keys](https://github.com/input-output-hk/hydra/issues/1141) in
+  relationship with Hydra Head protocol
+
+![Hydra voting UI](./img/hydra-voting.png) <small><center>Hydra Tallying UI</center></small>
+
+### zkSNARKs on Hydra
+
+The team started discussing with another Catalyst Fund10 project that
```

Comment:
Suggest to update this paragraph thus:
'The team has initiated discussions with another funded Catalyst Fund10 project, exploring the implementation of a [zero-knowledge proof system](https://projectcatalyst.io/funds/10/f10-development-and-infrastructure/a-zero-knowledge-proof-framework-for-cardano-based-on-hydra-and-zk-snarks) for Cardano, utilizing Hydra and ZK Snarks. At this early stage, there is minimal progress to display, but the team is eager to start the development of this promising tool.'